### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by imCaseSensitive**
Apologies for the mountain of commits.

This pull request covers the following:
- Change in opacity to users when selected for a hangout (in hangouts)
- Addition of an attendees modal that displays attendees for each event
- Other minor styling changes to things like background color and margins.

Warning: The key bug is back, and I haven't been able to find the cause of it in the code i've written here, although it is probably related to the UserList component.

### CODE REVIEW

File: components/User.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { View, Text, Image, StyleSheet, TouchableOpacity } from 'react-native'
 import Badges from './Badges'
 import { connect } from 'react-redux'
 import { sendHangoutRequest } from '../src/actions/hangouts'
+import { userInfo } from 'os'
```
Comment (bkhartmire) [inline components/User.tsx]: What's this?

[the remark this answers is not in the document]
Comment (imCaseSensitive) [reply components/User.tsx]: No idea what this is :O

[the remark this answers is not in the document]
Comment (imCaseSensitive) [reply components/User.tsx]: Deleted ^^

File: src/reducers/reducer.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ const initialState = {
   blockedByUsers: [],
   popupModal: false,
   showProfile: false,
+  showAttendees: false,
```
[the remark this answers is not in the document]
Comment (imCaseSensitive) [reply src/reducers/reducer.js]: triggerAttendees is a helper boolean for ordering the display of modals (2 modals cannot be active at once, and changing the state for showAttendees and showEvent in the same action was not effective.

File: views/EventModal.tsx [hunk 3]
```diff
@@ -55,6 +58,32 @@ interface Event {
 
 const EventModal: React.FunctionComponent<Props> = (props) => {
 
+    let attendeesButton
+
+    const openAttendees = () => {
+      props.triggerAttendees()
+      props.closeEvent()
+    }
+
```
[the remark this answers is not in the document]
Comment (imCaseSensitive) [reply views/EventModal.tsx]: Unfortunately, this is the only way I could find to chain modals.

**Pull request merged**